apps/coursition/src/components/forgotPassword.tsx

```diff
@@ -0,0 +1,56 @@
+'use client'
+
+import { Button, Input } from '@nmit-coursition/design-system'
+import * as React from 'react'
+import { useFormState } from 'react-dom'
+import { toast } from 'sonner'
+import { sendResetPassword } from '../app/actions'
+
+// biome-ignore lint/suspicious/noExplicitAny: <explanation>
```

Comment:
**blocker:** You should add explanation, I think it would be the same as I already have somewhere else in the codebase.